apps/emqx/test/emqx_common_test_helpers.erl

```diff
@@ -690,28 +693,38 @@ emqx_cluster(Specs0, CommonOpts) ->
     ]),
     %% Set the default node of the cluster:
     CoreNodes = [node_name(Name) || {{core, Name, _}, _} <- Specs],
-    JoinTo0 =
+    JoinTo =
         case CoreNodes of
             [First | _] -> First;
             _ -> undefined
         end,
-    JoinTo =
-        case maps:find(join_to, CommonOpts) of
-            {ok, true} -> JoinTo0;
-            {ok, JT} -> JT;
-            error -> JoinTo0
-        end,
```

Comment:
I think this should stay, otherwise we lose the ability to choose the node to join to?

Reply:
We shouldn't. `CommonOpts` [will be part of](https://github.com/emqx/emqx/blob/acd6e5635b6b8d8ea396a922d2e972338c8a020d/apps/emqx/test/emqx_common_test_helpers.erl#L1060-L1075) `Opts` [there](https://github.com/emqx/emqx/blob/acd6e5635b6b8d8ea396a922d2e972338c8a020d/apps/emqx/test/emqx_common_test_helpers.erl#L728) anyway, and they win over other (automatic) options in the end.

Reply:
True, forgot there are a bunch of `merge_opts` for that.  Ignore me.